uniffi_bindgen/src/bindings/kotlin/templates/Helpers.kt

```diff
@@ -41,7 +44,7 @@ internal class ConcurrentHandleMap<T>(
     fun insert(obj: T): Handle =
         lock.withLock {
             rightMap[obj] ?:
-                currentHandle.getAndAccumulate(stride) { a, b -> a + b }
+                currentHandle.getAndAdd(stride)
```

Comment:
The tests pass, but I'm not 100% sure I haven't subtly messed something up here with the `.also` and what-not...

Reply:
No, this is fine.

At some point, we'll need to start writing lower level tests for all this helper code.

Reply:
Indeed. I don't have a good idea of how to structure such tests though :-/